puppetserver-standalone/Dockerfile

```diff
@@ -44,4 +44,14 @@ EXPOSE 8140
 ENTRYPOINT ["dumb-init", "/docker-entrypoint.sh"]
 CMD ["foreground" ]
 
+HEALTHCHECK --interval=10s --timeout=10s --retries=90 CMD \
+  curl --fail -H 'Accept: pson' \
+  --resolve 'puppet:8140:127.0.0.1' \
+  --cert   $(puppet config print hostcert) \
+  --key    $(puppet config print hostprivkey) \
+  --cacert $(puppet config print localcacert) \
+  https://puppet:8140/production/status/test \
```

Comment:
That's a very good idea. I've updated the PR with a new `PUPPET_HEALTHCHECK_ENVIRONMENT` just to keep the namespace consistent.
Regarding the api request, I used an alias that always points to the latest version, instead of having the version hardcoded in it (`.../v3/...` ). I'll be happy to change it if you guys think it's more appropriate.